.github/workflows/publish-demos.yml

```diff
@@ -0,0 +1,218 @@
+          tar -xzf devextreme-dist-installer.tgz -C node_modules/devextreme-dist
+          mv node_modules/devextreme-dist/package/* node_modules/devextreme-dist
+          tar -xzf devextreme-angular-installer.tgz -C node_modules/devextreme-angular
+          mv node_modules/devextreme-angular/package/* node_modules/devextreme-angular
+          tar -xzf devextreme-react-installer.tgz -C node_modules/devextreme-react
+          mv node_modules/devextreme-react/package/* node_modules/devextreme-react
+          tar -xzf devextreme-vue-installer.tgz -C node_modules/devextreme-vue
+          mv node_modules/devextreme-vue/package/* node_modules/devextreme-vue
+
+      - name: Prepare configs
+        working-directory: apps/demos
+        run: npm run prepare-js
+      
+      - name: Copy metadata
+        working-directory: apps/demos
+        run: npm run make-demos-bundle -- --copy-metadata
+
+      - name: Prepare React demo bundles
+        working-directory: apps/demos
+        run: npm run make-demos-bundle -- --framework React
+
+      - name: Prepare Vue demo bundles
+        working-directory: apps/demos
+        run: npm run make-demos-bundle -- --framework Vue
+
+      - name: Save publish-demos directory for ES as an artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: publish-demos
+          path: ./apps/demos/publish-demos
+
+  publish-angular:
+    runs-on: ubuntu-latest
+    needs: [build-devextreme]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Check generated demos (1/10)
+            command: CONSTEL=1/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (2/10)
+            command: CONSTEL=2/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (3/10)
+            command: CONSTEL=3/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (4/10)
+            command: CONSTEL=4/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (5/10)
+            command: CONSTEL=5/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (6/10)
+            command: CONSTEL=6/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (7/10)
+            command: CONSTEL=7/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (8/10)
+            command: CONSTEL=8/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (9/10)
+            command: CONSTEL=9/10 npm run make-demos-bundle -- --framework Angular
+          - name: Check generated demos (10/10)
+            command: CONSTEL=10/10 npm run make-demos-bundle -- --framework Angular
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
Why not v4? We will get warnings because v3 use node16

Reply:
Just copy paste it from prev version, where 16 node was needed, seems like nothing stops us now for updating

Reply:
Actually, there is a bug related to upload-artifact@v4 
https://github.com/actions/upload-artifact/issues/478
Lead to failing CI
https://github.com/ivanblinov2k17/DevExtreme/actions/runs/9792470726/job/27038840580
Downgraded it from v4 to v3